unnecessary byte butter allocation

jaemyung opened the pull request:
In my opinion, repetitive byte buffer allocations inside while loop look unnecessary since we can reuse allocated buffer.

I started to learn c# recently, so it might be wrong due to lack of my understanding. It would be nice opportunity to learn things that I missed if this proposal is incorrect. Otherwise, I believe it could improve the example.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

Commit:
unnecessary byte butter allocation

In my opinion, repetitive byte buffer allocations inside while loop look unnecessary since we can reuse allocated buffer.

I started to learn c# recently, so it might be wrong due to lack of my understanding. It would be nice opportunity to learn things that I missed if this proposal is incorrect. Otherwise, I believe it could improve the example.

rpetrusha reviewed (changes requested):
Thank you for contributing to the dotnet/docs repo, @jaemyung. You're correct; since the buffer is already allocated, new allocations are unnecessary. There is, however, a Visual Basic example that has the same problem on line 33. If you could change the VB code as well, we'll merge your PR.

Commit:
remove unnecessary allocation for Visual Basic 

remove unnecessary allocation for Visual Basic as @rpetrusha suggested

simply compiled & tested (the change is obvious but just in case)

jaemyung commented:
updated for Visual Basic example as well!

rpetrusha reviewed (approved):
Thank you, @jaemyung, for making the additional change. We'll merge your PR now. Your changes should appear live on docs.microsoft.com in the next day or two.
